apm-sniffer/apm-sdk-plugin/armeria-0.85.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/armeria/define/Armeria086ClientInstrumentation.java

```diff
@@ -32,6 +32,7 @@
 public class Armeria086ClientInstrumentation extends ClassInstanceMethodsEnhancePluginDefine {
```

Comment:
This class name is not suitable anymore. Please consider to refactor?

Reply:
Sure, what about renaming to the lowest version that the instrumentation supports from? For example Armeria096ClientInstrumentation indicates that it supports 0.96+, and if it supports all future versions that would be best, if it doesn’t support from, for example 1.2.3, we will have another Armeria123...? 


But I’m recently consider to rewrite the plugin from 1.0.0 with a more recommended tracing solution in Armeria, decorator, but that’s another story.

Reply:
> For example Armeria096ClientInstrumentation indicates that it supports 0.96+

That totally works for me. 

> But I’m recently consider to rewrite the plugin from 1.0.0 with a more recommended tracing solution in Armeria, decorator, but that’s another story.

If that could replace some of the existing plugins, we could delete them later. It is not an issue.